packages/vue-dot/src/elements/FilterManager/FilterManager.vue

```diff
@@ -0,0 +1,121 @@
+<template>
+	<VMenu offset-y>
+		<template #activator="{ on, attrs }">
+			<VBtn
+				color="secondary"
+				dark
+				rounded
+				v-bind="attrs"
+				class="mr-2"
+				style="text-transform:lowercase;"
+				v-on="on"
+			>
+				{{ displayFiltersCount }}
+				<VIcon
+					color="white"
+					class="ml-2"
+					@click="resetFilters"
+				>
+					{{ closeIcon }}
+				</VIcon>
+			</VBtn>
+		</template>
+		<VList>
+			<VListItem
+				v-for="(item, index) in displayFiltersRow"
+				:key="index"
+			>
+				<VListItemTitle>
+					<div style="display: flex; align-items:center;">
+						<div style="width:300px;">
+							<div class="my-2">
+								{{ item.filter.fieldOptions.modalTitle }}
+							</div>
+							<!--<div class="my-2">
+								{{ `${item.filter.form.filter.value.value.length} ${item.filter.label} ${item.filter.form.filter.value.value.length > 1 ? 'sélectionnés' : 'sélectionné'}` }}
+							</div>-->
+						</div>
+						<div
+							class="mr-2"
+							@click="editFilters(item.index)"
+						>
+							<VIcon color="black">
+								{{ editIcon }}
+							</VIcon>
+						</div>
+						<div
+							class="mr-2"
+							@click="clearFiltersRow(item.index)"
+						>
+							<VIcon
+								color="black"
+								class=""
+							>
+								{{ deleteIcon }}
+							</VIcon>
+						</div>
+					</div>
+				</VListItemTitle>
+			</VListItem>
+		</VList>
+	</VMenu>
+</template>
+
+<script lang="ts">
+	import Vue, { PropType } from 'vue';
+	import Component, { mixins } from 'vue-class-component';
+
+	import { mdiCloseCircle } from '@mdi/js';
+	import { mdiWindowClose } from '@mdi/js';
+	import { mdiPencil } from '@mdi/js';
```

Comment:
```suggestion
	import { mdiCloseCircle, mdiWindowClose, mdiPencil } from '@mdi/js';
```
On peut grouper ces trois imports je pense !